### PR TITLE
[Student][E2E][MBL-13976]: Speculative fix for GradesE2ETest instability

### PR DESCRIPTION
**Pull request by joehoag**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/student/src/androidTest/java/com/instructure/student/ui/pages/CourseGradesPage.kt [hunk 1]
```diff
@@ -125,7 +125,7 @@ class CourseGradesPage : BasePage(R.id.courseGradesPage) {
         do {
             try {
                 gradeValue.check(matches(matcher))
-            } catch(e: NoMatchingViewException) {
+            } catch(e: Exception) {
```
Comment (joehoag) [inline apps/student/src/androidTest/java/com/instructure/student/ui/pages/CourseGradesPage.kt]: I told Trevor to us NoMatchingViewException, but it looks like that is not the only type of exception that can get thrown here.  So I dialed it back to plain old "Exception".

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021947&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by joehoag** [comment]
Thanks for the review!

**Pull request merged**